scripts/gather_version_artifacts.py

```diff
@@ -10,20 +10,12 @@
 # Before this script runs, the job downloads the artifacts into files with the following example structure:
 #
 #   versions_artifacts/
-#       3.7/
```

Comment:
Why did you delete it? The location of the artifacts wasn't changed.

Reply:
I misread the comment :-)